IBAnimatableApp/TransitionTableViewController.swift

```diff
@@ -12,16 +12,17 @@ class TransitionTableViewController: UITableViewController {
   // MARK: - Lifecycle
   override func viewDidLoad() {
     super.viewDidLoad()
-    
-    for type in iterateEnum(TransitionAnimationType) {
-      transitionAnimations.append(type)
-    }
+    transitionAnimations.append(.Fade)
```

Comment:
Let's think about this implementation here. We need to group them into sections. But how 🤔